microk8s-resources/actions/values.yaml

```diff
@@ -0,0 +1,344 @@
+# Default values for Falco.
+
+image:
+  registry: docker.io
+  repository: falcosecurity/falco
+  tag: 0.27.0
+  pullPolicy: IfNotPresent
+  pullSecrets: []
+
+docker:
+  enabled: true
+  socket: /var/run/docker.sock
+
+containerd:
+  enabled: true
+  socket: /run/containerd/containerd.sock

```

Comment:
How is this used? I am asking because the containerd socket in MicroK8s is not in this path.